packages/mgt-chat/src/components/ChatListHeader/ChatListHeader.tsx

```diff
@@ -47,20 +48,19 @@ export const ChatListHeader = (
   return (
     <div className={classes.headerContainer}>
       <div>
-        {buttonItems.map((buttonItem, index) => (
-          <Button key={index} className={classes.button} onClick={buttonItem.onClick}>
-            <div className={classes.buttonIcon}>
-              <Circle>{buttonItem.renderIcon()}</Circle>
-            </div>
-          </Button>
-        ))}
+        {props.isSignedIn === true &&
```

Comment:
`=== true` is redundant here and on line 63

Reply:
Without `=== true`, we are just verifying the property is set.